recipe/meta.yaml

```diff
@@ -24,12 +23,14 @@ requirements:
     - setuptools
     - numpy
     - fftw >=3.3.8
-    - liblapack >=3.8
+    - liblapack >=3.8  # [unix]
+    - libopenblas      # [win]
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - fftw >=3.3.8
-    - liblapack >=3.8
+    - liblapack >=3.8  # [unix]
+    - libopenblas      # [win]
```

Comment:
note: the fact that numpy does not automatically detect liblapack is a bug.

Reply:
I don't think this is the right solution. Using `liblapack` installed the netlib versions of blas and lapack. numpy works correctly with it in many other windows builds. What exactly is not working?

Reply:
numpy shoud automatically detect lapack when launched inside of anaconda. 

see https://github.com/numpy/numpy/commit/bc9dc649157dd426202c86d41ce50091243a88b6

I will try this fix, and then adjust if it doesn't work.

Reply:
If this package needs to depend on a newer version of numpy you can specify that. By default it builds against the oldest ABI compatible numpy it can to be as widely compatible as possible.